### ISSUE TITLE
Allow mocks to be assigned to vars

### ISSUE THREAD

**Issue by eyalroth** [paulbutcher/ScalaMock#123]
I really want to be able to assign a mock to a `var`.

Why can't I? because this will not compile:

```
var m = mock[MyClass]
(m.do _).expects()
```

I know of the workaround:
```
val workaround = m
(workaround.do _).expects()
```
But it's very... non aesthetic.

**Comment by paulbutcher** [comment]
This sounds very much like #62, which was fixed in ScalaMock 3.2. Which version of ScalaMock are you using?

Feel free to reopen if you believe that this is a different issue.

**Comment by eyalroth** [comment]
It was. I'm dumb. Thanks.
